rdrf/rdrf/templates/rdrf_cdes/base_questionnaire.html

```diff
@@ -30,10 +30,48 @@
                 table {
                     table-layout: fixed;
                 }
+
+		ul.errorlist {
+		    display: block;
+		    padding: 15px;
+		    margin-bottom: 20px;
+		    border: 1px solid transparent;
+		    border-radius: 4px;
+		    color: #a94442;
+		    background-color: #f2dede;
+		    border-color: #ebccd1;
+		}
             </style>
+
         {% endblock %}
         
     </head>
+    <script>
+      // scroll to any validation errors
+      $(document).ready(function() {
+
+      (function() {
```

Comment:
Yeak good call - Initially I had the body outside the ready call so was thinking i'd close off the variables later I moved it into the ready call which makes it unnecessary.